paddle/operators/activation_op.h

```diff
@@ -51,159 +61,210 @@ class ActivationGradKernel : public framework::OpKernel<T> {
     auto dx = framework::EigenVector<T>::Flatten(*dX);
     auto place = context.GetEigenDevice<Place>();
     Functor functor;
+    auto attrs = functor.GetAttrs();
+    for (auto& attr : attrs) {
+      *attr.second = context.Attr<float>(attr.first);
+    }
     functor(place, x, y, dy, dx);
   }
 };
 
+template <typename T>
+struct BaseActivationFunctor {
+  using ELEMENT_TYPE = T;
+
+  using AttrPair = std::vector<std::pair<const char*, float*>>;
```

Comment:
Why here attr is always float?  Attr has its own data type, and will be static_cast to activation functor data type